### PR TITLE
Boxstation Mapping Changes

### PR DESCRIPTION
**Pull request by Steelpoint**
This PR is intended as larger series of minor, and maybe major, changes to Boxstation. In the hope of improving it and warming up reception to this somewhat neglected map. 

(This PR is a WIP)

**I HIGHLY** welcome any **feedback** and **suggestions** for future changes or tweaks to the map in any capacity, one thing I've been told is that Box maint could be enlarged but I'm still thinking about how I'd approach that, if at all.

Below I'll list every single change I've effected, for transparency sake as well since its not unheard of minor things to be swinged through. Pictures will be included at the end of the more bigger changes.

I've currently gone through Security and some civilian areas. I'm now working my way around Engineering, Medical and Science. Maintenance is also on my agenda.

----------------

* Hydro dirt mounds in the Perma Brig have been replaced with proper hydro trays.
* Surgical tools in the 'prisoner transfer centre' are now inside a surgical dufflebag.
* Sergeant-at-Armsky has been posted to the armoury.
* External armoury now has a security camera placed there.
* HoS Office has been electrified and now has privacy shutters
* The oft unused 'cell 4' is now a communal holding cell (pic below)
* EVA has been reworked to be far more accessible in a emergency and now has additional space suits, eight up from four. Other misc supply count is akin to how it was prior. 

* Botany now has two public facing vendors for public use, a MegaSeed and NutriMax vendor.
* Science now has a public autolathe via the 'RnD' window.

* New fire safety closets and a few extra emergency lockers have been sprinkled around the main halls.
* Bar backroom now has a new Hooch Master Deluxe for mixing.

* Security entrance now has a window in between the doors.
* ORM is now public facing
* Two hand labblers were placed in science.

----------------

screenshot 2018-11-21 00 03 51
EVA

screenshot 2018-11-21 09 06 18
Botany

screenshot 2018-11-21 00 04 25
Public Autolathe

screenshot 2018-11-21 09 05 44
Security Entrance and Common Cell

screenshot 2018-11-21 09 06 04
ORM

:cl: Steelpoint
add: NTSS Boxstation has undergone several large and minor scale retrofits and refits by the Nanotrasen Corps of Engineers. 
tweak: Changes of note include a reworked EVA Storage, improved permanent confinement living area and a public autolathe. A full list of changes can be found on your local NT Git page.
/:cl:

### CODE REVIEW

**Comment by yoyobatty** [comment]
Reminds me of the edits we did on hippiestation to our hippiestation.dmm map, based on boxstation. I looked at meta for inspiration

**Comment by anconfuzedrock** [comment]
Yeah sure turn boxstatin into meta why don’t we, those life prisoners need light up hydro trays cuz they’re often too busy to check, Something something steelpoint buffing sec, conflicts based around lack of spacesuits are stupid and cargo should never consider buying more, it isn’t like metastation storage was built for highpop anyways.
Hundreds of assistant lives were ruined by having to ask a botanist for seeds, but fortunately their lives will be saved when we turn box into metastation and decide that, yes, autollathes were balanced around being public. Chemists always should have access to 50 large beakers,. Any assistant can own a 15 brute saw and multitool with a whole surgical set, spraycantide should require no effort, cargo must be one dimensional. Yes, maps should have no gameplay differences, only different locations of departments. Heaven forbid we make one thing harder to do on a certain map!

**Comment by Steelpoint** [comment]
Being sarcastic and acting retarded is not going to get you far. If you disagree on certain changes, please describe them constructively.

**Comment by 81Denton** [comment]
tl;dr
at least format your feedback so it's readable

**Comment by ExcessiveUseOfCobblestone** [comment]
> a third of these are sec buffs

hehehehehehe

I will say I agree with unloved in the fact that I don't want this to be a meta clone. I liked the look of older EVA and absolutely despise the resource supplying such as extra vendors (more stranges!) and a free autolathe.

**Comment by MMMiracles** [comment]
public autolathes make me irrationally angry

**Comment by Steelpoint** [comment]
At the very least the vendors are not free for the crew to print off. I also thought it'd be another minor money sink for the one person who wants to grow fruit and did not spawn as a Botanist.

The public autolathe idea was more due to every other station using one, hence why I thought it'd be an acceptable change and was one suggested to me. If public autolathes really are a issue that allow too much good gear to get into peoples hands, I'd suggest tackling the idea by making public variants of autolathes more restricted in items they can make or adding some expensive monetary cost to vending items. 

The EVA change was inspired by me wishing that EVA was more accessible during an emergency, the extra EVA suits were more so to bring the storage closer in line with the other stations. But if scarcity of suits is desired I can reduce the number.

The security changes are more due to the fact I personally notice those things more so, and I can hardly call most of em real buffs aside finally giving box armsky.

**Comment by ExcessiveUseOfCobblestone** [comment]
adding security cameras and adding defense mechanisms to hos office are straight buffs however warranted they may be.

**Comment by Steelpoint** [comment]
Its more a correction considering not only does every other station variant have those, but that we USED to have those on Box in the past but they were, for reasons beyond me, removed.

**Comment by yoyobatty** [comment]
That snowballed quickly, I like the changes though, might carry them over to hippie :)

**Comment by anconfuzedrock** [comment]
Alright steel, I’ll say the exact same things twice.
-prisoners don’t need hydro trays, they have little else to do but watch plants and don’t deserve more luxury than the public garden. Plus prisoners will just smash the machines.
-eva has 8 suits on meta because it was designed for highpop. Box’s comparatively low suit count can lead to interesting conflict and gives cargo reason to buy eva crates or mining to buy hardsuits(it also used to have 6, 2 being for command, but they were removed because it was argued that it wasn’t necessary since all heads but the hop had suits now and, at the time, the hop could get a suit from mining.)
-I see no reason why assistants need instant access to all of the seeds botany has. They can just ask a botanist, or do what assistants do if there are none (or just “ai open” or ask hop or one of many legal ways)
-autolathes were balanced around being private, but people love the convenience without regarding the flaws.  Not every map needs to be the exact same. With private autolathes, cargo had more work. I felt some value to my large beakers as chemist, though I could get more if I needed to. Free 15 brute weapons weren’t meant to be handed out to assistants, and spray cans are incredibly easy to tide with. Spray paint brig windows, ruin the halls faster than a janiborg can mop, and dear god the stuns. As much as I use them, I shouldn’t have gotten away with murder by spray painting a sec officer in the face. Other ghetto assistant weaponry pale in comparison (ied spear maybe, but lances were never intended as throwing weapons and here we are now with the op trash.) .357 loaders cost several tc because they weren’t something any job could buy by the barrel. At this point on meta detectives have no reason not to immediately mod their gun. If anything, meta should be the outlier,, Not box, but it can be an outlier because
-not every map needs to be exactly the same, just with different department locations. Nobody whines when things are easier on one map, but they do if things are harder. The only job with risk of death, even, is shaft miner. Causing people to have to put in effort to get their rewards can’t be a bad thing. Vacant commisaries somewhat stump creativity in that it means anyone making a shop is almost certain to build there, and doing otherwise would be a waste, though creative. Same applies to most rooms on non box maps- the maint bar on meta is fully decorated, well, in general meta is, so you have less room for your own decor. I get the feeling people mostly like meta because at face value it looks pretty and it has a lot of free stuff.(which it specifically only has because it was made as the first highpop map) That doesn’t mean every map needs to have all the cool stuff meta does.

**Comment by ExcessiveUseOfCobblestone** [comment]
> At the very least the vendors are not free for the crew to print off. I also thought it'd be another minor money sink for the one person who wants to grow fruit and did not spawn as a Botanist.

True but also it means double FREE seeds, double FREE nutrients, and most importantly double stranges for botanists.

> The public autolathe idea was more due to every other station using one, hence why I thought it'd be an acceptable change and was one suggested to me. If public autolathes really are a issue that allow too much good gear to get into peoples hands, I'd suggest tackling the idea by making public variants of autolathes more restricted in items they can make or adding some expensive monetary cost to vending items.

It is restricted by being in Cargo only, which you have just removed. Other maps are dumb for this I absolutely can't stand it.

> The EVA change was inspired by me wishing that EVA was more accessible during an emergency, the extra EVA suits were more so to bring the storage closer in line with the other stations. But if scarcity of suits is desired I can reduce the number.

okay, I just dislike the look of it. I don't have much consideration of the number of suits.

> Its more a correction considering not only does every other station variant have those, but that we USED to have those on Box in the past but they were, for reasons beyond me, removed.

so it's a buff for consistency lol

**Comment by Steelpoint** [comment]
The above feedback is something I can understand, I'll wait longer for more feedback however before I make any changes.

---

> -eva has 8 suits on meta because it was designed for highpop. Box’s comparatively low suit count can lead to interesting conflict and gives cargo reason to buy eva crates or mining to buy hardsuits

I've yet to see any 'interesting conflict' arise from this, at most its just a case of losing out on a EVA suit. I've also never seen cargo order anything that isn't a SWAT suit for space proof protection but that's beside the point and a economy issue. 

> -I see no reason why assistants need instant access to all of the seeds botany has. They can just ask a botanist, or do what assistants do if there are none (or just “ai open” or ask hop or one of many legal ways)

Assistants still need to spend their money on the seeds and items, money they have in very limited supply mind you. 

> -autolathes -snip-

Traitors have been breaking into cargo and looting dozens of revolver/stechkin ammo for years, public or private. At least with the more public autolathes you are likely to be seen printing the ammo, although in the current meta no one really gives a shit. The other issues I can see as being valid, but I feel a better solution would be forcing people to spend money to print items so at least there is more reason to save money. 

> -not every map needs to be exactly the same, just with different department locations. Nobody whines when things are easier on one map, but they do if things are harder. -snip-

I disagree that my changes are morphing this map into a meta clone, I just have a very symmetrical design scheme and that tends to mimic meta to a degree. Imo I feel one of the bigger draws to meta is the very large and secretive maintenance it has which also helps antag play in some regard. 

-----

> True but also it means double FREE seeds, double FREE nutrients, and most importantly double stranges for botanists.

I could remove some of the more pressing items from the vendor (stranges) and slightly decrease overall numbers of seeds.

**Comment by Steelpoint** [comment]
I've pushed some changes.

* New fire safety closets and a few extra emergency lockers have been sprinkled around the main halls. 
* Bar backroom now has a new Hooch Master Deluxe for mixing.

**Comment by anconfuzedrock** [comment]
If you don’t see it, I don’t have much else to say, really. People will give at least something more of a shit about ammo when people break into cargo for it. You don’t see much conflict over suits because yes swat suits are too cheap, and because every map but box has too many suits for it to matter.
It takes minimal effort for an assistant to steal cash (or they could play engineer) for any other job, they now have basically free access to half the benefits of botany.
Until autolathe prices are actually added, DON’T BALANCE MAPS AROUND THEM!
I don’t know what the hell a symmetrical design scheme is supposed to mean in this context, but meta’s maint helps antag play in the same way that a baby walker helps a baby walk. Limits creativity (why make a place to hide the bodies or convince people to stay outta your stash in the library quiet room when you can weld into one of your many flavors of majnt loot room?) And doesn’t teach you a thing. At the least, meta’s maint is large because it was designed for a particularly large amount of people.

**Comment by Steelpoint** [comment]
If an assistant wants to grown plants and food I don't want to discourage em, since most assistant players hardly do anything of worth or contribution to begin with. 

I might end up tossing the autolathe but I'll wait for more feedback before I go removing things from this PR. 

The maint issue is a fair point, but antags like cult are very reliant on a larger maint to help keep their activities hidden for longer. There are very few locations on the map, maint wise, that can offer some degree of stealth.

**Comment by anconfuzedrock** [comment]
Ye steel, so they can make em. If cults couldn’t just plop down in maint bar, they’d do thingsxlike kidnap the virologist and settle there, convert a whole small department so they can hide out in there, use the emgineer to build a new room, etc. I’d go into detail but this is kind of irrelevant and mostly just me shitting on maint

**Comment by ExcessiveUseOfCobblestone** [comment]
If an assistant wants to plant then they can go botanist instead of rolling for antag maint access from the getgo :)

**Comment by epochayur** [comment]
8 suits seems like overkill

**Comment by 81Denton** [comment]
To the people complaining about maint and ""free"" items: Whenever I ask people why they like Delta and Meta so much, 90% they say it's A) their expansive maint that allows for lots of creativity, whether if it's for renovating/construction or being an antag. B) is that jobs have plenty of resources, since ordering basic neccessities is fun for exactly no-one.

I'd advise anyone to actually ask players for their opinions and start mapping themselves instead of just bikeshedding in the comments section.

**Comment by anconfuzedrock** [comment]
> To the people complaining about maint and ""free"" items: Whenever I ask people why they like Delta and Meta so much, 90% they say it's A) their expansive maint that allows for lots of creativity, whether if it's for renovating/construction or being an antag. B) is that jobs have plenty of resources, since ordering basic neccessities from cargo is fun for exactly no-one.

Ok denton. that's why people like meta. That's why they like META. if they want those things, they can play METASTATION. It is a completely different station. I like the pyro in tf2, and I like the heavy's sandvich, but giving the pyro the sandvich would not be a good idea. Even if it was a good idea, then I would just have heavy and burn-heavy. no pyro. what if i just want pyro? heavy players who want burn heavy can just get the huo-long heater. There are other classes with healing-others capabilities that they can play too, but there is only one flame-focused airblast class.
Or I could just argue that the expansive maint and free stuff just actually lower variety and creativity, only seeming nice at face value. but I've already done that. If cargo doesn't need to order stuff, what does it do?

I was going to make my own map until people talked me out of it. I still am planning on doing so eventually.

**Comment by Dorsisdwarf** [comment]
I don't like how the EVA main room is now behind blast doors and rwalls instead of windows. 

That little nubbin of window outside botany is now pointless and ugly, remove it.

Thats all the opinion I really have here.

**Comment by Tlaltecuhtli** [comment]
move orm

**Comment by Tlaltecuhtli** [comment]
add hand labelers to science

**Comment by Nickkoranda** [comment]
> Ok denton. that's why people like meta. That's why they like META. if they want those things, they can play METASTATION. It is a completely different station.

Most of the time [on bagil at least] it is because of map rotation that we end up being on box. Most of the time people try to convince admins to change the map, if not admins themselves hosting a vote for a map change/changing it themselves.
Oh, and every map vote box gets maybe 1-5 votes while meta gets 20+, that should say something.

**Comment by nicbn** [comment]
I dislike the EVA change, EVA worked perfectly well for ages and now it looks empty and ugly.

**Comment by Nickkoranda** [comment]
That being said I like these changes a lot. IMO box should've been heavily reworked to become modern or get removed, it is a legacy map that nobody [until now] has supported.

**Comment by anconfuzedrock** [comment]
Ok Denton, let's say that "more fun to play and designed better" is constructive enough that I can't argue otherwise. if that's the case, then they can play meta and delta. Turning box into "meta but less fancy" has no merits what so fucking ever.Give me a fucking reason why we can't have maps with significant differences, Give me a fucking reason why not one single station can lack a feature that metastation has. 

Just because at face value a public autolathe makes it easier to get your circular saw as assistant, doesn't mean that this is good for variety and challenge. Even though pre built rooms in maint can make it easier to set up as antag or otherwise, it stunts creativity by nearly forcing people to use said rooms. While metastation and delta might look a bit nicer than box at roundstart, this just means that there is less for someone to decorate on meta and it can just as well be argued that box fits better as a "metal death trap". 

Maybe you completely disagree with me! Maybe you don't care what my opinion is! Maybe you just want a map that feels good at face value! If that's the case, then just vote against box! You're not going to like it much after this, but people who liked box before might now hate it. There are three other maps to vote for if you're so opposed to box. a few rounds a day are going to be rotated to box- this is inevitable. Let's say it's 20% of rounds, maybe that's too low, maybe that's too high. I'm fairly damn sure that at least 20% of people like boxstation.

Box isn't "supported" because the people who like box like it the way it is. Most of the problems with box are entirely subjective, and aren't direct issues (which still exist on delta and meta-doors to virology come to mind). Stop talking like box is equivalent to ministation or omega, where there were actually direct needs for updates, and no direct updates. Calling the subjectively flawed minimalist designs of boxstation "objective issues" and "unsupported" is just fucking bullshit.

I was going to argue against "having restricted autolathes would work better on higher rp servers" but then I realised I don't need to, because autolathes are very fucking similar to chem dispensers, which we refuse to make public. We get by with chemists just fine, and if anything chemicals are more important than autolathe supplies, which there are usually plenty of spares of. We have gotten by with the non public autolathe before, since for several years boxstation was the only map.

Steelpoint, you're just doing what you think people want, but you're making a mistake. This is just going to make boxstation completely bland, without even a vague "niche" like delta has as a highpop ripoff of metastation. Why even keep box around after this? I don't think anyone who likes these changes likes box at all, but even after these changes, they'll undeniably prefer metastation just the same. This will be worse than removing box, people who liked box might see less reason to, while people who hate box aren't going to change their map preferences.

**Comment by coiax** [comment]
I think the implication that Box's uniqueness is defined by its restriction of autolathes is silly.

**Comment by anconfuzedrock** [comment]
That is one of the many things. and yes I can see why it sounds stupid, but it also sounds stupid that assistants can mass print cans that can vandalise hallways in seconds, stun security officers, and make every window on the station opaque. There is no reason to add a public one to box, and I am beyond strongly opposed to the idea that every map needs these "quality of life" additions. I don't care if I _sound_ silly, I haven't felt this strongly about a change all year, for what it does and for its implications.

**Comment by Steelpoint** [comment]
The issues with Boxstation are pronounced enough that I'm concerned it might eventually lose its default position on the server, or even see removal, due to it being a neglected map to a degree. Box regularly loses out against other maps in votes in game and akin.

I'm not aiming to make this a clone of Meta, Delta or any other map, but to simply try and improve areas of it and bring it somewhat in line with the expectations of other maps that people have.

**Comment by coiax** [comment]
@anconfuzedrock You know what would be a weird meet-in-the-middle? Make a public autolathe that charges $50( or some non-trivial amount) per use, with the money going to the Cargo Budget.

Real departments will have no trouble accessing money, but assistants will then start mugging the clown in order to get that $$$.

**Comment by Steelpoint** [comment]
Ironically mapping guides tend to point to Metastation as a good, even golden, standard of maintenance design.

**Comment by anconfuzedrock** [comment]
I don't think box is neglected, per se, it simply doesn't need much change. I think that the specific expectations you are bringing it in line with are things that shouldn't be expectations at all- the implication that maps need a public autolathe (convenience isn't always better!) and the random free seed vendors (anyone who needs such a variety of seeds that they can't ask botany for some seeds should just play botanist, these are only in because meta had them and hyperactive assistant mains only see the way things immediately benefit them)
until that happens coiax, this isn't a good idea, and even then, I don't like the idea of an assistant looting a braindead doctor for money and then just buying spray cans or whatever. it needs regulation, not costs. 
I don't care what the mapping guides say steel. Especially since the most notable mapping guide was literally written by metacide (guess which station he made) I don't think it matters at all what they say.

**Comment by coiax** [comment]
What about making a sort of Box-hopline style queue with shutters, moving the autolathe to be behind it, starting down by default, which Cargo can open to make their autolathe public access.
Thus they can open it for public demand, or keep it shut.

**Comment by anconfuzedrock** [comment]
That's a good idea coiax, but I don't see why compromises need to be made here, there already is a compromise- some maps have public autolathes, other maps do not. Not that that's a bad idea.

**Comment by Steelpoint** [comment]
New changes.

* Security entrance now has a window in between the doors.
* ORM is now public facing
* Two hand labblers were placed in science.
* Removed the pocking out glass window by botany

screenshot 2018-11-21 09 05 44

screenshot 2018-11-21 09 06 04

screenshot 2018-11-21 09 06 18

**Comment by Steelpoint** [comment]
Again, I note that the seed vendors are not free for the public and the botanists still need to pony up for the more coveted items.

I'm a consistently security player, and I have never in all my years seen assistants rushing me with spray cans or circular saws. At best spears and other makeshift gear. This greytide apocalypse, while is here to a extent with round start rushings of the bridge and captains quarters, is not as doom and gloom as you profess.

**Comment by Steelpoint** [comment]
Fair point. Nonetheless maintenance is a contentious issue, however I'm more saying this in theory as I've not yet made any changes to box maint.

**Comment by coiax** [comment]
I did a thing that I was talking about doing. And it's unique to Box! None of the other stations have an overly elaborate auto lathe queue system.

**Comment by anconfuzedrock** [comment]
I don't know steelpoint, I've had to wipe my windows several times as warden because assistants paint it black, warden being my second most played job. And I know from experience as an assistant that you can stun a sec officer using a spray can and absolutely demolish him(maybe a fluke? they're still amazingly useful if you just want to make a janitor cry though.) Assistants don't usually make circular saws, but I don't think they should be able to regardless. I miss feeling value to my large beakers as chemist, when I had to go out to ask cargo for more. now a small beaker is basically garbage and I'd never consider making a grenade with them. 
As for the seeds costing money, again I stress that money is negligible for 99% of jobs (And sometimes for assistants if there's a braindead around) but I think I should put my stress on the other fact, that if a person needs that many more seeds than the ones they can just ask hydro for or find at the garden, then that person can just play botanist. any job with a decent amount of money (aka any job) will be able to do the majority of botany's job with those vendors.
I just don't understand the trays in permabrig please explain to me I just don't get it they're permabrig prisoners why do they deserve full hydro trays and why do they need them, it won't stop them from breaking out windows, the trays are destructible themselves, they don't need the warning lights since they usually have spare time to watch plants.

Everything about the autolathe rubs me the wrong way. With it, any job has access to a full set of surgical equipment and multitools are everywhere. I think that multitools being a bit rarer than other tools is nice, it makes them feel special and it's more interesting to hack a door when all you have are cable cutters, since they're more dangerous and permanent. the instant access to tools is also contrary to them costing money in the toolvends. In general, tool storage is outclassed by the autolathe, poor vendomat (Assistants definitely shouldn't be able to make rcds either, especially since they were buffed to eat raw glass as fuel.) I'd compare asking cargo to make you an rcd or spray can to asking the hop for access to engineering or something, you'll probably get a yes, but you should have to ask. I don't even think it's something that's a matter of compromise, though I think coiax's idea of a simple is good and I'd like that, I don't even consider that as a compromise, that's just a good idea. I just don't think every single map having a public autolathe is a good thing. its the perfect place to have variety! 

I'm not going to say anything about maint, since I've said it before. I repeat myself too often but I'm sure you understood me the first time about pre-built rooms crippling creativity, and box being perfect as a simple blank slate.

Also, I'm interested in where exactly the new fire safety and emergency oxygen closets are. I think I said this but I _really_ think that fire safety closets would just be a good fit in the two emergency storage closets, maybe one at atmos and sec. that's the only one I don't have several paragraphs to argue for I just think it'd fit nicely with them all being in specific closets in maint so

**Comment by imsxz** [comment]
the thing i see complained about most on box is the lack of a public auto lathe, its why most people dislike playing on it is what I gather from personal observation. sure you might enjoy your xtra hardmode SS13 veteran map but the majority of the players are newer and enjoy playing with the luxuries of meta/delta. if nobody enjoys playing a map except for a few people, perhaps things need to be babied down a bit.

**Comment by Tlaltecuhtli** [comment]
@Steelpoint having the vendors there looks meh how about putting only the seed vendor in the public garden instead

**Comment by Bawhoppen** [comment]
The box in Boxstation must have always been an abbreviation of Hugbox.

I don't get why we have to try and make everything sleek and hyper-streamlined for every map. 
Isn't the whole point of ss13 that you're supposed to be on a barebones metal-deathtrap of a station? 

Also, public autolathes are really silly and should be removed. We survived for years with having to ask/break into cargo for autolathe stuff, and now they're finally being added to every map. All they do is just exacerbate the attitude of the playerbase to have less one-on-one interaction than it already does.

(Some of these changes are good though. Box has needed a few legitimate updates for quite some time now.)

**Comment by Tlaltecuhtli** [comment]
firelocks are badly mapped in box here is an example:
http://take.ms/6LqB2
just put 1 firelock under a door

**Comment by Vidiveni** [comment]
I like the public ORM, so it will stop being tedious for science to upgrade it. 

The lack of public autolathe made it rewarding to build or move one, but it's a minor issue for me. What I've seen more complaints about is "boring maints". Basically too few interesting items or rooms to find in maint. 

P.S. please add a box of pill bottles to chem locker (unless I remember wrong and there is one already).

**Comment by anconfuzedrock** [comment]
How is maint on other maps less boring? Wasting time welding down a wall for a syringe gun isn’t interesting, definitely less so than having to actually steal one. I say deltastation maint is boring because the pre-built rage cage sucks the fun out of making your own.

**Comment by Mark9013100** [comment]
Is there any reason to change the ORM's location? Most people just take mats from a techfab/protolathe most of the time.

**Comment by Tlaltecuhtli** [comment]
@Mark9013100 you have to touch it to upgrade when you dont have brped

**Comment by Mark9013100** [comment]
Fair point

**Comment by Barhandar** [comment]
@Tlaltecuhtli box has roundstart silver ore in the dock, if you have rped on box you're a borg or miners are incompetent and don't deserve upgrades.

**Comment by ExcessiveUseOfCobblestone** [comment]
Can we cap the different changes please? I don't want to discuss X then have someone post "b-b-b-but Y is good" and force a PR merge/close because of that. Honestly if you want to trash all but ironically your sec changes I think this would be a great PR.

**Comment by improvedname** [comment]
Dont forget that autolathes equals 60 brute ranged weaponry.

**Comment by anconfuzedrock** [comment]
nicbn over here pulling words right outta my asshole and converting them into legible format that makes me rethink my life choices

**Comment by Steelpoint** [comment]
If certain features are "too" controversial I'll split them off at a later point.

**Comment by subject217** [comment]
man that's a whole lot of words to complain about the consistently most outdated and least liked map getting better

**Comment by Tlaltecuhtli** [comment]
why isnt there a :fire: emoji

**Comment by Steelpoint** [comment]
if I do any maintenance changes I'll likely reserve that for a future PR.

**Comment by anconfuzedrock** [comment]
Probably because deltastation was made by a literal nazi and the only thing it has going for it is that its maint tunnels are excessively wide. About as accurate as subjects drivel is.

**Comment by Steelpoint** [comment]
If EVA is too big I could reduce it in size and use the leftover space for maintenance, I like the current setup but I'm open to altering if it needed.

I could relocate the autolathe to cargo, or I can remove it if the head honchos don't want a public lathe on Box (and I think most of the lathe issues could be resolved with a economy tweak or making the public lathes not have as many options).

I'd like to keep the seed venders but I could relocate just the seed vendor to the public garden and remove the other vendor if requested. 

I think the ORM being public facing is fine, it helps make upgrades a bit earlier if needed and most people can already access the minerals via their departmental techfabs.

**Comment by Steelpoint** [comment]
I'll hold off on any more changes in this PR and save other things for a follow up PR.

**Comment by nicbn** [comment]
I'd like to see EVA have its former low suit amount.

Remove the public lathe for now, this should be included in another PR.

Yep, relocate the vendors to the garden.

ORM is fine.

**Comment by anconfuzedrock** [comment]
Ok, you can make eva easier to break into ironically by removing the windows, since now you only have to hack one door in maint.
But whyyyy do prisoners need hydropooniiics trays I just don't understaaaaand what's the poiiiiiiint like whyyyyyyy what was wrong with diiiiirt

**Comment by necromanceranne** [comment]
All of these have been needed changes to box to modernize it's design. Very epic.

**Comment by ExcessiveUseOfCobblestone** [comment]
Box is Notre Dame, the other maps are Modern Offices.

Box has history and intricacy (barring the stupid maint de-evolution), other maps just brag about how big/how streamlined they are

**Comment by subject217** [comment]
notre dame doesn't look like a turd though

**Comment by ExcessiveUseOfCobblestone** [comment]
Box and Best both begin with B's, coincidence? No, just alliteration.

**Comment by epochayur** [comment]
box is de_dust2, meta is cs_office

**Comment by subject217** [comment]
similarly dust2 is a dull overused map and office is a classic for the ages

**Comment by anconfuzedrock** [comment]
Subject box runs like once a fucking week, meta is 2fort, box is viaduct. It’s not hard to see why people like meta with its mindless fun but if you want something balanced and functional you should look at viaduct

**Comment by optimumtact** [comment]
The other maps are literally hugboxed to shit to make people like them more, we lost our last actually talented mapper who cared more about the game experience than just handing people stuff on a plate when ikky up and took off.

**Comment by Steelpoint** [comment]
It'd be easier to discuss a mapping standard on what we could expect from maps and what is, and is not, on offer to people.

**Comment by Cyberboss** [comment]
Forcing mapdiffbot

**Comment by Cyberboss** [comment]
Conflicts

**Comment by Steelpoint** [comment]
oranges seems against the PR

**Comment by coiax** [comment]
Break your changes into smaller pieces, definitely should be able to get some of them in.

**Comment by Steelpoint** [comment]
I'll do that
